src/main/sensors/gyro.c

```diff
@@ -169,7 +169,7 @@ static void gyroInitLowpassFilterLpf(gyroSensor_t *gyroSensor, int slot, int typ
 #define GYRO_OVERFLOW_TRIGGER_THRESHOLD 31980  // 97.5% full scale (1950dps for 2000dps gyro)
 #define GYRO_OVERFLOW_RESET_THRESHOLD 30340    // 92.5% full scale (1850dps for 2000dps gyro)
 
-PG_REGISTER_WITH_RESET_TEMPLATE(gyroConfig_t, gyroConfig, PG_GYRO_CONFIG, 3);
+PG_REGISTER_WITH_RESET_TEMPLATE(gyroConfig_t, gyroConfig, PG_GYRO_CONFIG, 4);
```

Comment:
This is needed after removing the defaults?

Reply:
The defaults aren't actually why I have incremented the version, it is because I have reordered and optimised the parameter group.

(Even though incrementing the parameter group version is a way to make sure users get the new defaults, since changed defaults will only be used when the parameter group is reset)